libyul/backends/evm/EVMDialect.cpp

```diff
@@ -55,7 +55,7 @@ void visitArguments(
 	for (auto const& arg: _call.arguments | ranges::views::reverse)
 		_visitExpression(arg);
 
-	_assembly.setSourceLocation(_call.location);
+	_assembly.setSourceLocation(_call.debugData->location);
```

Comment:
Especially as this file just accesses it directly, why not use `locationOf()` here too? (Also this could crash if there's no debugdata)